### PR TITLE
Add --skip-requirements to install.sh and enable SUPPORTS_SSE42 override

### PR DESCRIPTION
**Pull request by erinaceous**
I've discovered that the check for SSE4.2 added in #361 fails on virtualization solutions other than KVM - possibly what the author of #1782 is having trouble with too.

Trying to install Sentry on a virtual machine running on a VMWare ESXi host ("ESXi 6.5 and later (VM version 13)"), it reports running on processors that _definitely_ [support SSE4.2](https://ark.intel.com/content/www/us/en/ark/products/120495/intel-xeon-gold-6154-processor-24-75m-cache-3-00-ghz.html):

```cat /proc/cpuinfo
model name	: Intel(R) Xeon(R) Gold 6154 CPU @ 3.00GHz
```

But this is not reported in their `flags`.

In this case, this is likely to do with our VM cluster having old servers which really don't have SSE4.2 and so [VWMare targets the lowest possible instruction set](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vcenterhost.doc/GUID-8B226625-4923-410C-B7AF-51BCD2806A3B.html) -- or, there's just a bug or a misconfigured option which means SSE4.2 isn't reported for CPUs which really do support it.

Obviously the right solution is "fix that underlying problem", but that's outside of our control (virtual machines managed by a central IT department, long lead time on updating the VMWare environment, etc).  

Luckily, I was able to run the executable mentioned in [this comment](https://github.com/ClickHouse/ClickHouse/issues/20#issuecomment-226853025) on a ClickHouse issue, which verified that yes, our virtual machine _does_ support SSE4.2, it just doesn't report it.

Bypassing the check in `check-minimum-requirements.sh`, I was able to install Sentry just fine on the VM.

Decided to clean it up and make it part of the install script - in two ways - being able to override `SUPPORTS_SSE42`, so you could run `SUPPORTS_SSE42=1 ./install.sh`, but also more generally skip requirements checking with an "I know, my set-up is weird, carry on anyway" flag to the script, `--skip-requirements`

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Comment by aminvakil** [comment]
What do you think about changing this to `--skip-sse42-requirements` ?

**Comment by ethanhs** [comment]
@erinaceous are you still interested in working on this PR? Totally fine if not but just wanted to check in where we're at here :)

**Comment by erinaceous** [comment]
> What do you think about changing this to `--skip-sse42-requirements` ?

Yep, noted the thumbs up on that comment from all you maintainers, changed it to be more specific :+1:

File: install/parse-cli.sh [hunk 1]
```diff
@@ -17,7 +17,11 @@ Options:
  --skip-commit-check    Skip the check for the latest commit when on the master
                           branch of a \`self-hosted\` Git working copy.
  --skip-user-creation   Skip the initial user creation prompt (ideal for non-
-                          interactive installs).
+                          	interactive installs).
+ --skip-sse42-requirements
+			Skip checking that your environment meets the
+			  requirements to run Sentry. Only do this if you know
```
Comment (ethanhs) [inline install/parse-cli.sh:23]:
```suggestion
			  requirements to run Clickhouse on x86. Only do this if you know
```
The real need of this is clickhouse uses SSE 4.2 on x86 (not on ARM)

**Review by ethanhs — APPROVED** [review]
Thank you for updating this! Just one suggestion.

**Pull request merged**